### PR TITLE
Aeh profile post upd

### PR DESCRIPTION
**Pull request by aidan-hubley**
**Changes**
- Updated all icons and added color to toolbars (for Post and new OwnedPost)
- New OwnedPost (part of post.js) 
    - Used on Profile page
    - Removed Profile Pic and Post owner
    - Added:
        - Delete Post btn
        - Change Settings btn (turn off interactions, turn of commenting)
        - Edit Post (title, description, type)
- Updated 'Circle' Icon (far right white circle) to a new simplistic icon to match vibe 

**Notes**
- Overall update to padding and spacing
- I'm not passionate about my design revisions, willing to discuss them
- Mostly looking for ways to optimize code and differences on iOS

**YouTrack Link**
[PC-129](https://prayercircle.youtrack.cloud/agiles/147-2/current?issue=PC-129)

### CODE REVIEW

File: components/Post.js [hunk 1]
```diff
@@ -119,7 +120,7 @@ export const Post = (post) => {
 		if (!post.end) {
```
Comment (dev-arekusandoru) [inline components/Post.js]: i think the purpose of this initially was to draw dividers under each post. as far as I know we don't need any of this code anymore

**Review by dev-arekusandoru — CHANGES_REQUESTED** [review]
Overall I really like the design changes implemented in this PR. The icons and colors all look great, and i think the 'edited' text is a nice addition. A few things I think could be improved:
1. The new `OwnedPost` component is a good idea but I dont think it needs to be it's own component, It's really a 'hide/show this element' difference. I'd suggest adding those elements into the original Post component, and then look for prop `post.owned==true` to make them visible as well as hide whatever elements.
2. On the OwnedPost component you moved the 'time since created' text into the circle. I think this causes the icon to look a little crowed and the text out of place, since it isn't there on the main post. I'd like to brainstorm moving this to a different location on the post

Like a I said above, im a big fan of these changes, just some code-specific things that can help us out in the future

**Comment by aidan-hubley** [comment]
Overall I agree with the feedback and would like to discuss different approaches before moving on. 

- 1. I am unsure how easy it is to edit front end code (thinking developer settings in a browser). If this is an accessible tool for malicious users, simply hiding powerful post buttons will not be enough. 
   - Example: 'Delete post' button is hidden on the feed page, but a malicious user unhides it and is able to delete posts directly from the feed. 

- 2. The visual differences in post component feed and profile is note-able. (Profile image, profile name, 'age of post' relocation, 'interaction count', and toolbar differences) this feels valuable enough (particularly the potential relocation of 'post age') to justify a different component. 

-3. I also am not a fan of the current location for 'post age' in OwnedPost component. Any thoughts on where it could go? (An effort was made to scrap repeated elements to give more space to post details) aka with post owner being removed I see no great location for post age... Open to suggestions

**Comment by dev-arekusandoru** [comment]
@aidan-hubley some valid points, i'll do my best to address them as thoroughly as i can

1. This should not present any problems. While it technically would be possible for someone to edit the client-side code to gain access to the buttons, they would be prevented from doing anything malicious by the database rules(not yet in place).
2.  I do agree there are a few visual differences and possible relocations, but I don't think this warrants the entire duplication of the component. We are duplicating a lot of code with regard to animations and overall post structure that is unnecessary, and by adding a few logic operators to show/hide certain elements we will make the code more streamlined and make future development easier(a simple `owner` prop as opposed to an entirely new element)
3. I haven't given too much thought to this, but I suggest we open discussion with the entire team to discuss what we think should happen here.

File: components/Post.js (hunk 1 above)
Comment (aidan-hubley) [reply components/Post.js]: Which piece is not needed anymore?

Comment (dev-arekusandoru) [reply components/Post.js]: the entire `if(!post.end)` and its contents

**Comment by aidan-hubley** [comment]
@dev-arekusandoru consolidated to single component.

A post now needs 'owned', 'interaction', and 'edited' attributes

**Review by dev-arekusandoru — APPROVED** [review]

**Pull request merged**